src/api/app/views/webui2/webui/project/list.html.haml

```diff
@@ -0,0 +1,51 @@
+:ruby
+  @pagetitle = 'Public Projects'
+.card
+  - if @important_projects.present?
+    .card-body
+      %h3 Main Projects
+      .table-responsive
+        %table.table.table-sm.table-striped.table-bordered
+          %thead
+            %tr
+              %th Name
+              %th Title
+          %tbody
+            - @important_projects.each do |project|
+              %tr
+                %td
+                  = link_to(project[0], action: :show, project: project[0])
+                %td
+                  #{project[1]}
+  .card-body
+    %h3= @pagetitle
+    .d-inline
+      %p
+        - if @show_all
+          = link_to(projects_path) do
+            %i.fas.fa-toggle-on.text-primary
+            Exclude #{::Configuration.unlisted_projects_filter_description}
+        - else
+          = link_to(projects_path(show_all: true)) do
+            %i.fas.fa-toggle-off.text-primary
+            Include #{::Configuration.unlisted_projects_filter_description}
+        - unless User.current.is_nobody?
+          %span.float-right
+            = link_to(new_project_path) do
+              %i.fas.fa-plus-circle.text-primary
+              Create new project
```

Comment:
This should be `Add New Project` to be consistent with other Bootstrap pages (as a comparison, subprojects is using `Add New Subproject`)